balance_service/enforcement/usage_enforcement.py

```diff
@@ -166,6 +166,7 @@ def check_usage_against_allocation(self, data):
                     project_id=lease["project_id"],
                     user_id=lease["user_id"],
                     start_date=lease["start_date"],
+                    name=lease["name"],
```

Comment:
The root problem is that we don't have reservation.id at this point I take it?

Reply:
yes. reservation id is assigned after the enforcement checks.